Debian/bin/lib/Cyrus/Docker/Command/build.pm

```diff
@@ -0,0 +1,48 @@
+use v5.36.0;
+
+package Cyrus::Docker::Command::build;
+use Cyrus::Docker -command;
+
+use Process::Status;
+
+sub execute ($self, $opt, $args) {
+  my $root = "/srv/cyrus-imapd";
+  chdir $root or die "can't chdir to $root: $!";
+
+  my $version = `./tools/git-version.sh`;
+  Process::Status->assert_ok("determining git version");
+
+  chomp $version;
+
+  say "building cyrusversion $version";
+
+  system('git clean -dfx');
```

Comment:
Ahhhhhhhhhhhhhh (this already hosed things for me once).

We should figure out the point of cyr build. On a fresh install, clean isn't necessary. And if you're making small changes to the C code, you don't want a complete rebuild. Guess it comes down to how you intend this to be used.

Reply:
I've removed this for now.  This line is just pulled in from previous code and actually won't usually matter.  We'll bring it back as an option later or something.

Reply:
Thanks